test/ReaderIO.ts

```diff
@@ -0,0 +1,101 @@
+import * as assert from 'assert'
+import { pipe } from 'fp-ts/lib/pipeable'
+import * as R from 'fp-ts/lib/Reader'
+import * as IO from 'fp-ts/lib/IO'
+import * as _ from '../src/ReaderIO'
+
+describe('ReaderIO', () => {
```

Comment:
It seems that Jest was completely unaware of the `ReaderIO` module, so as soon as `test/ReaderTaskEither.ts` references it the coverage checker fails.